### DISCUSSION TITLE
how to enable insights for a public repo

### DISCUSSION THREAD

**Discussion by pplmx** [community/community#35028] [Enterprise]
As title.

**Comment by pplmx** [comment]
> 4. Under "Member organization permissions", select or unselect Allow members to view dependency insights.

CANNOT FOUND "Member organization permissions"

**Comment by jfullstackdev** [comment]
unfortunately, the documentation is for Enterprise Cloud

![temptemp](https://user-images.githubusercontent.com/47092464/194105246-a031c62c-906f-420f-ba9a-45ae4088fbe9.png)

**Reply by pplmx** [reply]
I find some repos can display "insights", somes not. I don't why.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @pplmx ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

**Reply by pplmx** [reply]
Thanks a lot.
